models.py

```diff
@@ -23,7 +23,7 @@ class ClientOrganization(Base):
         server_default=text("nextval("
                             "'client_organization_client_organization_id_seq'"
                             "::regclass)"))
-    client_organization_code = Column(Text, nullable=False)
+    client_organization_code = Column(Text)
```

Comment:
In the future we might want to set it to not null, but for now this is arbitrary and can't be set from the GUI, therefore needs to be nullable.